### ISSUE TITLE
Meepo's 

### ISSUE THREAD

**Issue by zapxcero** [jeffhill/Dota2#332]
### Cosmetic Item Name

Colossal Crystal Chorus

### Description

Meepo's original aghs upgrade provides additional clone (total of 5 meepo) and the cosmetic item's sound fx will be used each time Meepo casts poof. Now, when you choose the +1 clone lv25 talent (total of 5 meepo), you still don't hear the custom poof sound fx. The only way is to buy aghs instead which has no relation to the current aghs upgrade. Thanks! Idk why there's a match id needed for cosmetic bug item maybe make it optional?

### Example Match ID

0

**Comment by OMEGALULW** [comment]
>Idk why there's a match id needed for cosmetic bug item maybe make it optional?

IT LITERALLY IS. Simply delete it. Edit your post and remove ### Example Matchb ID and ### Screenshots you idiot.

**Comment by Naaveen-Kumar** [comment]
> IT LITERALLY IS. Simply delete it. Edit your post and remove ### Example Match ID and ### Screenshots you idiot. Also, make your title descriptive. No one knows what "Meepo's" is referring to.

@OMEGALULW  You can try being nice. Not everyone knows how to describe an issue in a platform like GitHub. This might be new for the person.

**Comment by OMEGALULW** [comment]
> @OMEGALULW You can try being nice. Not everyone knows how to describe an issue in a platform like GitHub. This might be new for the person.

No. Valve apologists have gone way out of their way to defend Valve lately. I'm not going to be nice to all these Valve-supporting shitheads who don't want Dota 9 to become Dota 2 again.

**Comment by zapxcero** [comment]
I have fixed it. This is the first time I'm using this issue thing. The guy above is just sad. Go back to facebook.

**Comment by OMEGALULW** [comment]
>I have fixed it. This is the first time I'm using this issue thing. The guy above is just sad. Go back to facebook.

It's funny how a redditor is telling someone who's never used facebook to go back to facebook.

**Comment by NMOTH** [comment]
Would you mind adding [QoL]  to the title :)
